extensions/cornerstone-3d/src/Viewport/Overlays/ViewportOverlay.tsx

```diff
@@ -42,6 +42,11 @@ function CornerstoneViewportOverlay({
         upper
       );
 
+      // Don't set VOI if already set
+      if (voi.windowCenter !== null || voi.windowWidth !== null) {
```

Comment:
Surely this needs to be an `&&`? I know its an edge case and shouldn't be hit, but if one of them is not set we should set the voi right?